acme/acme/standalone_test.py

```diff
@@ -146,19 +145,22 @@ def tearDown(self):
 
     def test_it(self):
         max_attempts = 5
-        while max_attempts:
-            max_attempts -= 1
+        for attempt in range(max_attempts):
             try:
                 cert = crypto_util.probe_sni(
                     b'localhost', b'0.0.0.0', self.port)
-            except errors.Error:
-                self.assertTrue(max_attempts > 0, "Timeout!")
-                time.sleep(1)  # wait until thread starts
-            else:
                 self.assertEqual(jose.ComparableX509(cert),
```

Comment:
I think it'd be good to keep this in the `else:` block as it minimizes the amount of code in the `try` block which is largely the purpose of the `try`, `except`, `else` pattern in Python.